uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-requests.js

```diff
@@ -189,7 +189,7 @@ define([
      */
     vm.getUserNameByID = function (id) {
       var user = _.find(vm.filteredUsers, function (contact) {
-        return contact.contact_id === id;
+        return contact.id === id;
```

Comment:
in the PR description you have this

> In few places id is used in place contact_id because the leaveManagees does not return contact_id.

but, why? id is used in place of contact_id on what object? always `contact`? Why does it happen in "few places"? Did it work at all before?